src/libsyntax/ext/tt/macro_rules.rs

```diff
@@ -453,7 +453,10 @@ fn is_in_follow(cx: &ExtCtxt, tok: &Token, frag: &str) -> bool {
             // harmless
             true
         },
-        _ => cx.bug(format!("unrecognized builtin nonterminal {}",
-                            frag).as_slice()),
+        _ => {
+            // invalid fragment specifier. Just pass it along, it will be
+            // caught in macro_parser::parse_nt()
+            true
```

Comment:
This seems reasonable to me, should add in a comment in `parse_nt` that this bit of code depends on it rejecting invalid fragment specifiers.